app/src/main/java/in/testpress/testpress/ui/MainActivity.java

```diff
@@ -574,6 +575,11 @@ protected void setEmptyText(final int title, final int description, final int le
     @Override
     public void onResume() {
         super.onResume();
+
+        if(isVerandaLearningApp()){
+            showTermsAndConditionsIfNotAgreed();
```

Comment:
startActivity(TermsAndConditionActivity.Companion.createIntent(MainActivity.this));